packages/geo/src/lib/catalog/catalog-browser/catalog-browser-layer.component.ts

```diff
@@ -10,9 +10,14 @@ import { CatalogItemLayer } from '../shared';
 @Component({
   selector: 'igo-catalog-browser-layer',
   templateUrl: './catalog-browser-layer.component.html',
+  styleUrls: ['./catalog-browser-layer.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CatalogBrowserLayerComponent {
+export class CatalogBrowserLayerComponent implements OnInit {
+
+  public tooltip: string;
```

Comment:
Not being an observable, are you sure the tooltip is updated properly?

Reply:
fixed